frontend/public/locales/zh-CN/application.json

```diff
@@ -69,5 +69,10 @@
     "pipelineInfo": "数据管道信息",
     "serverEdp": "服务器端点",
     "serverDomain": "服务器域"
+  },
+  "delete": {
+    "title": "删除应用程序",
+    "tip1": "您确定要删除该应用程序 ",
```

Comment:
Could we merge tip1 and tip2 and add placeholders for variable strings?

for example, https://github.com/awslabs/clickstream-analytics-on-aws/pull/1024/files#diff-a3cca7198752321283623781b842e9c87d263f0ba76c83514735cf73c3d6b97fR315